src/OrbitGl/App.cpp

```diff
@@ -1766,11 +1766,13 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::RetrieveModuleAndLoadSymbols(
   load_result.Then(main_thread_executor_, [this, metric = std::move(metric),
                                            module_id](const ErrorMessageOr<void>& result) mutable {
     if (result.has_error()) {
+      modules_with_symbol_loading_error_.emplace(module_id);
       metric.SetStatusCode(OrbitLogEvent::INTERNAL_ERROR);
     }
     symbols_currently_loading_.erase(module_id);
   });
 
+  modules_with_symbol_loading_error_.erase(module_id);
```

Comment:
Shouldn't this be done at the beginning of the function?

Reply:
I moved it up